### PR TITLE
Run3-hcx254 Add test to compare cell parameters for HCAL geometry by DDD vs DD4Hep

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Add test to compare cell parameters for HCAL geometry by DDD vs DD4Hep

#### PR validation:

Make a comparison between the 2 versions and find identical results for HB

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special. Will be useful for testing ECAL geometry

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/HcalAlgo/data/cmsExtendedGeometry2021.xml [hunk 1]
```diff
@@ -0,0 +1,272 @@
+<?xml version="1.0"?>
+<DDDefinition>
+  <open_geometry/>
+  <close_geometry/>
+
+  <IncludeSection>
+    <Include ref="Geometry/CMSCommonData/data/materials.xml"/>
```
Comment (cvuosalo) [inline Geometry/HcalAlgo/data/cmsExtendedGeometry2021.xml:7]:
```suggestion
    <Include ref="Geometry/CMSCommonData/data/materials/2021/v1/materials.xml"/>
```

Comment (bsunanda) [reply Geometry/HcalAlgo/data/cmsExtendedGeometry2021.xml:7]: Change as suggested

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by cvuosalo** [comment]
+1

**Comment by silviodonato** [comment]
+1

**Pull request merged**